### PR TITLE
Test that we handle stored mx_last_room_id correctly

### PR DESCRIPTION
**Pull request by richvdh**
* Do we show the last room correctly on resume?
 * Do we show the home page if we have no last room?
 * If we follow a login link, and re-log-in, do we correctly vape the mx_last_room_id (#4283)

This will fail until the corresponding matrix-react-sdk PR lands.

### CODE REVIEW

File: test/app-tests/loading.js [hunk 1]
```diff
@@ -271,9 +245,30 @@ describe('loading:', function () {
             localStorage.setItem("mx_is_url", "http://localhost" );
             localStorage.setItem("mx_access_token", "access_token");
             localStorage.setItem("mx_user_id", "@me:localhost");
+            localStorage.setItem("mx_last_room_id", "!last_room:id");
```
Comment (lukebarnard1) [inline test/app-tests/loading.js]: I'm not sure I like the fact that these tests rely on each other... Could this be moved to the start of the next test?

Comment (richvdh) [inline test/app-tests/loading.js]: this is a `beforeEach`, not a test?

Comment (lukebarnard1) [inline test/app-tests/loading.js]: Oh! That's fine then :)

**Comment by richvdh** [comment]
> also, the tests are passing on travis....

Ah yes. That's because of the magic where we use the same-named branch of react-sdk.

**Pull request merged**